src/ftpclass.cc

```diff
@@ -2295,6 +2339,7 @@ int   Ftp::Do()
 	 if(res==-1 && errno!=EINPROGRESS)
 	 {
 	    saved_errno=errno;
+            LogError(0,"connect: numeric error code: %u",saved_errno);
```

Comment:
add the numeric code to the next message, if really needed

Reply:
I removed that one.